app/models/manageiq/providers/cloud_manager/provision_workflow.rb

```diff
@@ -90,6 +88,19 @@ def allowed_ci(ci, relats, filtered_ids = nil)
     super(ci, relats, sources, filtered_ids)
   end
 
+  def availability_zone_to_cloud_network(src)
```

Comment:
there seems to be a mapping, other way around in https://github.com/Ladas/manageiq-providers-amazon/blob/6b6cb06d25bd5d38e1c700fd34fcd87a6efec824/app/models/manageiq/providers/amazon/cloud_manager/provision_workflow.rb#L75

Anyway. This seems like Amazon specific code, so this should belong to amazon/cloud_manager/provision_workflow.rb. @bronaghs or do we have similar behavior for e.g. azure?

Reply:
Ladas, that's what https://github.com/ManageIQ/manageiq-providers-amazon/pull/380 is for.